### DISCUSSION TITLE
Usage Questions

### DISCUSSION THREAD

**Discussion by lightingman117** [ep1cman/unifi-protect-backup#85] [Q&A] [answered]
Novice here.

A few questions.

1) How do I know when it is done backing up (new install)?
2) (In Unraid) docker CLI I can't seem to run any commands, what do I do?
3) Seems to use a constant 2% CPU; not sure if it is doing something or just checking drive/api for new files?
4) is there a way to tell it to grab continuous clips not just events?

**Comment by ep1cman** [comment]
1) if you check the log it will show how long the download queue is 
2) not quite sure what you mean by this, could you elaborate?
3) I suspect the initial backup will show quite high usage for a while, while it is downloading every clip unless you told it not to. Some constant CPU load is also to be expected because the application is always listening for new events
4) This tool isn't really intended to save recordings of the camera feed, there are loads of tools that can do that via RTSP, this is specifically for backing up the event clips.

**Comment by lightingman117** [comment]
1. Thanks, how do I? (I think this is related to 2.)
2. See image. ![image](https://user-images.githubusercontent.com/54555519/232119868-a1b9ee04-7e20-4dc5-8461-2377b47e7d3c.png)

3. Understood. How does it download the clips? [meaning, isn't the UNVR device API handling the requests and only serving up the files for your script to download (simply store to drive) shouldn't most of the CPU be 'elsewhere'?
4. I see. So given your explanation of 3, I was thinking it would/could tell [UNVR to stitch the high-res (.ubv) recordings together from a camera](https://durdle.com/2022/01/22/extract-unifi-protect-video/) and transcode into .mp4 clips and that would make it capable of exporting entire camera's daily footage. Makes sense tho. I guess this is a feature request ;)

**Reply by ep1cman** [reply, marked as answer]
The tool works by talking to the UniFi protect API which has a function to generate a MP4 file between two time stamps. It never sees or touches uvb files. So yes most of the heavy lifting of the transcoding is done by your NVR but it's not uncommon for the first backup to contain tens of thousands of clips, so a continuous 2% CPU load isn't unexpected.

In unraid have you tried clicking the container icon and opening the log? You shouldn't need to be in the console to run the application since it auto starts when the container starts. The only time you might need to be in the console is to generate an rclone config file for backup to cloud storage (it comes with a default config to backup locally to the unraid server)

**Comment by lightingman117** [comment]
I'm an idiot. Thank you.

It's chugging through files...hundreds a minute.
2% load makes total sense now.
(I was thinking it was idle since the backup folder was already at 60GB and had clips in every folder for each camera).

~150 remaining :) [already been going since 10am (3hrs ago) WOOT!!]

**Comment by lightingman117** [comment]
With the download cue now at zero the CPU usage is also at zero :)

![image](https://user-images.githubusercontent.com/54555519/232124349-8ecdd395-3408-44f2-aaf3-a0ddb23a26cf.png)
